### PR TITLE
Gives extradimensional chap weapon a unique gimmick.

### PR DESCRIPTION
**Pull request by TechnoAlchemisto**
Force will randomly change from anywhere between 0 - 30.

I'm not entirely sure this was the best way to do this so if you know a better way uh.. let me know.

### CODE REVIEW

File: code/game/objects/items/weapons/holy_weapons.dm [hunk 1]
```diff
@@ -361,3 +361,14 @@
...
+
+
+/obj/item/weapon/nullrod/claymore/multiverse/New() //No copypaste here, no sir.
+	..()
+	START_PROCESSING(SSobj, src)
+
+/obj/item/weapon/nullrod/claymore/multiverse/Destroy()
+	STOP_PROCESSING(SSobj, src)
+	return ..()
+/obj/item/weapon/nullrod/claymore/multiverse/process()
```
Comment (RemieRichards) [inline code/game/objects/items/weapons/holy_weapons.dm]: no, don't do this.
just have every ```attack()``` change the value for the next attack.

**Comment by PKPenguin321** [comment]
30 tho?

**Comment by optimumtact** [comment]
10/10 coding techno

**Comment by TechnoAlchemisto** [comment]
this one time in grade school i took a web design class

**Comment by TechnoAlchemisto** [comment]
I mean, it works. winky face

i'm gonna try to figure out how to make it work remie's way give me an hour

**Comment by Incoming5643** [comment]
Keep in mind that a force rand(0,30) weapon is still on average a force 15 weapon

**Comment by TechnoAlchemisto** [comment]
Yeah, I know. Chappie weapons are (generally) force 18.

**Comment by optimumtact** [comment]
caring about the average force on rand() is just silly, it doesn't follow a normal distribution?

**Comment by RemieRichards** [comment]
no, it doesn't, but since the some of 0 to 30 is 465, and there are 31 numbers with 0 to 30, the average/median *IS* 15.

**Comment by Incoming5643** [comment]
@optimumtact No but it does regress towards the mean, the average number of hits to a knockdown will be similar to a force 15 weapon (7 hits before armor).

It's not a problem, it's just a neat math fact.

**Comment by TechnoAlchemisto** [comment]
Thanks for the help oranges

**Pull request merged**